students/ador_yano/Lesson02/DrawGrids.py

```diff
@@ -0,0 +1,52 @@
+# DrawGrids.py implements the Lesson 2 assignment from UWPCE Python Programming
```

Comment:
note that python modules (python files) should follow snake_case convention -> `draw_grids.py`

Reply:
Thanks for the feedback, Natasha. I'll study the python style guidelines.